### DISCUSSION TITLE
You are a legend

### DISCUSSION THREAD

**Discussion by ryanleecode** [saasykits/nextjs-sessionauth-template#54] [General]
This template is AMAZING! It's so much better than all the overpriced, useless "boilerplates" I've been seeing on Twitter lately. I spent a whole week struggling with crappy auth solutions before finding this gem.

Thank you so much for this incredible contribution, @iamtouha!

**Comment by iamtouha** [comment]
Hi @ryanleecode ,

It means a lot to know that this template made a positive impact for you. Thank you very much for your support and nice words!
